### PR TITLE
EVG-209: remove unused push time from models

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

**Comment by tychoish** [comment]
I think this field was always empty, but I'm a little worried about the version history drawer.

File: rest/model/task.go [hunk 1]
```diff
@@ -18,7 +18,6 @@ type APITask struct {
 	Id                 APIString        `json:"task_id"`
 	CreateTime         APITime          `json:"create_time"`
 	DispatchTime       APITime          `json:"dispatch_time"`
-	PushTime           APITime          `json:"push_time"`
```
Comment (richardsamuels) [inline rest/model/task.go]: Should we keep this in the API model, and just copy the CreateTime value to PushTime, since they're almost the same?

Comment (richardsamuels) [reply rest/model/task.go]: Furthermore, frontend js is using a push_time field on some objects. I can't determine what type they are though.

Comment (tychoish) [reply rest/model/task.go]: this value is populated from other sources, I think, in the front end code, and is just poor naming on the front end side.

Comment (richardsamuels) [reply rest/model/task.go]: Can you also purge push_time from the smoke test data?

**Pull request merged**